docs/server-non-voting-member.rst

```diff
@@ -0,0 +1,149 @@
+.. _server-non-voting-member:
```

Comment:
@gyuho @jpbetz shall we just use learner instead of non-voting member?

Reply:
sgtm

Reply:
@xiang90 Already merged but let me change it :)